ui/raidboss/data/04-sb/raid/o12s.js

```diff
@@ -57,10 +151,10 @@ export default {
       netRegexJa: NetRegexes.gainsEffect({ target: 'オメガ', effectId: '67E', capture: false }),
       netRegexCn: NetRegexes.gainsEffect({ target: '欧米茄', effectId: '67E', capture: false }),
       netRegexKo: NetRegexes.gainsEffect({ target: '오메가', effectId: '67E', capture: false }),
-      infoText: (data, _, output) => output.text(),
+      alertText: (data, _, output) => output.text(),
       outputStrings: {
         text: {
-          en: 'Keep Bosses Apart',
+          en: 'Move Bosses Apart',
```

Comment:
Should we clear the below translations?

Reply:
I think this is probably fine as-is.  Some of the below translations already look like they say this.

I wonder if this is a case where we need a `// FIXME` sort of support in the tools, where you can say "this translation is still good enough to keep, but somebody needs to look at it again."

Reply:
ja and cn are okay to keep, the verbs in the sentences has already the meaning of "pull the boss" or "move the boss to somewhere".